meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6/0001-board-apalis_imx6-sync-mmc-devices-with-linux.patch

```diff
@@ -0,0 +1,122 @@
+From c91baafe13de7bf2f6e882c021916587f3e58cbc Mon Sep 17 00:00:00 2001
+From: Michael Scott <mike@foundries.io>
+Date: Wed, 13 Jan 2021 13:38:01 -0800
+Subject: [PATCH] board: apalis_imx6: sync mmc devices with linux
```

Comment:
@vanmaegima @mike-scott wonder if here we should instead go with the generic solution proposed by @igoropaniuk at http://u-boot.10912.n7.nabble.com/RFC-PATCH-v2-1-2-optee-obtain-emmc-rpmb-info-from-dt-td438580.html (https://github.com/OP-TEE/optee_os/issues/4343), as I'm sure this will bite us again with another hardware later on.

That way we can have one generic patch in u-boot (+ board specific dts changes) and remove the apalis changes.